yarn-project/acir-simulator/src/avm/avm_memory_types.ts

```diff
@@ -237,6 +237,11 @@ export class TaggedMemory {
     return this._mem.slice(offset, offset + size);
   }
 
+  public getSliceAs<T>(offset: number, size: number): T[] {
```

Comment:
This is used when getting an array of MemoryType.Field, but casting them to be Fr